docs/gpa.md

```diff
@@ -194,28 +194,26 @@ The information included in the notice of intended procurement covers all the in
 
 </td>
       </tr>
-<!--
       <tr>
         <td>VII:4</td>
         <td>Procuring entities are encouraged to publish in the appropriate paper or electronic medium listed in Appendix III as early as possible in each fiscal year a notice regarding their future procurement plans (hereinafter referred to as "notice of planned procurement"). The notice of planned procurement should include the subject-matter of the procurement and the planned date of the publication of the notice of intended procurement.</td>
         <td markdown=1>
 
-* 
+* Enter *the subject-matter of the procurement* in `planning/rationale`
+* Enter *the planned date of the publication of the notice of intended procurement* in `tender/tenderPeriod/startDate`
```

Comment:
I agree with your suggestion to map to `tender.communication.futureNoticeDate`. `tenderPeriod` has the semantics of "submission period", but the date of publication of the notice might precede the submission period.

Reply:
I don't remember suggesting that, but I agree with myself!

Done

Reply:
In the PR's description :)